src/Bicep.LangServer.UnitTests/Handlers/BicepDeploymentParametersHandlerTests.cs

```diff
@@ -483,6 +483,7 @@ public async Task Handle_WithParameterOfTypeObjectAndNoDefaultValue_ShouldReturn
         public async Task Handle_ParameterWithDefaultValuesOfTypeExpression_ShouldReturnUpdatedDeploymentParametersWithIsExpressionSetToTrue()
         {
             var bicepFileContents = @"param location string = resourceGroup().location
+param deploymentLocation string = 'deploy-${resourceGroup().location}'
```

Comment:
nit: Can you use slightly complicated example here?
param dataFactoryName string = 'datafactory${uniqueString(resourceGroup().id)}'

Updated test with above change:
```
        [TestMethod]
        public async Task Handle_ParameterWithDefaultValuesOfTypeExpression_ShouldReturnUpdatedDeploymentParametersWithIsExpressionSetToTrue()
        {
            var bicepFileContents = @"param location string = resourceGroup().location
param dataFactoryName string = 'datafactory${uniqueString(resourceGroup().id)}'
param policyDefinitionId string = resourceId('Microsoft.Network/virtualNetworks/subnets', 'virtualNetworkName_var', 'subnet1Name')
resource blueprintName_policyArtifact 'Microsoft.Blueprint/blueprints/artifacts@2018-11-01-preview' = {
  name: 'name/policyArtifact'
  kind: 'policyAssignment'
  location: location
  properties: {
    policyDefinitionId: policyDefinitionId
  }
}";
            var template = @"{
  ""$schema"": ""https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#"",
  ""contentVersion"": ""1.0.0.0"",
  ""metadata"": {
    ""_generator"": {
      ""name"": ""bicep"",
      ""version"": ""0.13.189.62659"",
      ""templateHash"": ""13569275663664769498""
    }
  },
  ""parameters"": {
    ""location"": {
      ""type"": ""string"",
      ""defaultValue"": ""[resourceGroup().location]""
    },
    ""dataFactoryName"": {
      ""type"": ""string"",
      ""defaultValue"": ""[format('datafactory{0}', uniqueString(resourceGroup().id))]""
    },
    ""policyDefinitionId"": {
      ""type"": ""string"",
      ""defaultValue"": ""[resourceId('Microsoft.Network/virtualNetworks/subnets', 'virtualNetworkName_var', 'subnet1Name')]""
    }
  },
  ""resources"": [
    {
      ""type"": ""Microsoft.Blueprint/blueprints/artifacts"",
      ""apiVersion"": ""2018-11-01-preview"",
      ""name"": ""name/policyArtifact"",
      ""kind"": ""policyAssignment"",
      ""location"": ""[parameters('location')]"",
      ""properties"": {
        ""policyDefinitionId"": ""[parameters('policyDefinitionId')]""
      }
    }
  ]
}";
            var bicepFilePath = FileHelper.SaveResultFile(TestContext, "input.bicep", bicepFileContents);
            var bicepDeploymentParametersHandler = GetBicepDeploymentParametersHandler(bicepFilePath, bicepFileContents);

            var result = await bicepDeploymentParametersHandler.Handle(bicepFilePath, string.Empty, template, CancellationToken.None);

            result.deploymentParameters.Should().SatisfyRespectively(
                updatedParam =>
                {
                    updatedParam.name.Should().Be("location");
                    updatedParam.value.Should().Be("resourceGroup().location");
                    updatedParam.isMissingParam.Should().BeFalse();
                    updatedParam.isExpression.Should().BeTrue();
                    updatedParam.isSecure.Should().BeFalse();
                },
                updatedParam =>
                {
                    updatedParam.name.Should().Be("dataFactoryName");
                    updatedParam.value.Should().Be("format('datafactory{0}', uniqueString(resourceGroup().id))");
                    updatedParam.isMissingParam.Should().BeFalse();
                    updatedParam.isExpression.Should().BeTrue();
                    updatedParam.isSecure.Should().BeFalse();
                },
                updatedParam =>
                {
                    updatedParam.name.Should().Be("policyDefinitionId");
                    updatedParam.value.Should().Be("resourceId('Microsoft.Network/virtualNetworks/subnets', 'virtualNetworkName_var', 'subnet1Name')");
                    updatedParam.isMissingParam.Should().BeFalse();
                    updatedParam.isExpression.Should().BeTrue();
                    updatedParam.isSecure.Should().BeFalse();
                });
            result.errorMessage.Should().BeNull();
        }
```